lib/Doctrine/ORM/Configuration.php

```diff
@@ -33,7 +33,7 @@
  * Configuration container for all configuration options of Doctrine.
  * It combines all configuration options from DBAL & ORM.
  *
- * Internal note: When adding a new configuration option just write a getter/setter pair.
+ * {@internal When adding a new configuration option just write a getter/setter pair. }}
```

Comment:
We reverted these because IDEs started marking entire classes and methods as internal (warnings everywhere). How does this appear now in something like PHPStorm?

Reply:
I believe it's related to this https://github.com/Elgg/Elgg/issues/7714
PHPStorm show it strike through only when `@internal` (first version in phpdoc reference) is given :)

Reply:
![image](https://user-images.githubusercontent.com/8947920/34649735-0bf45180-f3b5-11e7-9382-3777d9567cb6.png)

Reply:
Configuration is definitely not an internal class (regardless how it looks in PhpStorm), the note is only meant to be a hint for anyone adding new options.

Reply:
Right, the class itself is not marked internal, just the text for adding the new options.

Reply:
The problem is just when the class is used - does the IDE still strike-through the name?

Reply:
With PhpStorm 2017.3.1 It's only strike-through in this case
```
/**
 * Configuration container for all configuration options of Doctrine.
 * It combines all configuration options from DBAL & ORM.
 *
 * {@internal When adding a new configuration option just write a getter/setter pair. }}
 *
 * @internal
 */
```

If anything else comes after the second `@internal` in the example, it's not striked-through anymore

Reply:
Verified locally 👍